README.md

```diff
@@ -1,2 +1,34 @@
 # comp110-worksheet-5
 Base repository for COMP110 worksheet 5
+
+(a)
+	Iterates through the list searching for duplicates
+
+(b)
+	Because when i and j are in different positions on the same list it returns true
```

Comment:
?

The answer I was looking for is that you have two nested loops over n elements -- hence `O(n*n) = O(n^2)`